README.md

```diff
@@ -1,2 +1,5 @@
-# hw01
-Repository for homework 01
+# Qitian Hu hw01
+* Hi! I am **Qitian** or **Jason**. A first year in the *college*, potentially majoring in ***philosophy and econ***. This my github homepage [Qitian's Github Page](https://www.google.com "Qitian's homepage") . 
+* I am relatively familar with **Markdown and github**, and the hardest part is actually to *fork the repository*. With the help from Deb, I am able to do so.
+
+![Screen Shot 2019-04-05 at 1.45.20 PM](my_image.png)
```

Comment:
For some reason, I don't see this alt-text rendering on the image. You could play around with [different ways of rendering](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) if you wanted to.